pyproject.toml

```diff
@@ -109,12 +109,11 @@ select = [
     "RUF015", # access first ele in constant time
     "RUF016", # type error non-integer index
     "RUF017",
-    "TRY200",
```

Comment:
We removed `TRY200`, and are instead recommending `B904`, which is an identical rule with a better implementation. So here, I removed `TRY200` and un-removed `B904` (above).